src/systemd/sysusers.d/cockpit-ws.conf

```diff
@@ -0,0 +1 @@
+u cockpit-wsinstance - "User for cockpit-ws instances" -
```

Comment:
Why is the filename different from the username?  That seems strange to me...

Reply:
Indeed, that's fallout from the original PR version having both users, but we eliminated cockpit-ws in the meantime. I renamed it.